spring-boot-3-demo-app/src/test/java/io/github/mfvanek/spring/boot3/test/controllers/TimeControllerTest.java

```diff
@@ -70,6 +81,20 @@ void cleanUpDatabase() {
     @SneakyThrows
     @Test
     void spanShouldBeReportedInLogs(@Nonnull final CapturedOutput output) {
+        final String zoneNames = TimeZone.getDefault().getID();
+        final LocalDateTime localDateTimeNow = LocalDateTime.now(ZoneId.systemDefault()).minusDays(1);
+        final ParsedDateTime parsedDateTime = new ParsedDateTime(
+            localDateTimeNow.getYear(),
```

Comment:
I would add the appropriate constructor/static method to create ParsedDateTime from LocalDateTime.